pkg/imported/l10n/en-us.yaml

```diff
@@ -15,6 +15,8 @@ imported:
     basics: Basics
     upgrade: Upgrade Strategy
     networking: Networking
+    registries: Registries
+    clusterMembers: Member Roles
```

Comment:
Looks like this got accidentally pulled in from a rebase. I'd like to remove this in a separate pr given the time and how often I'm having to re-run flaky tests.

Reply:
Removed